components/x-topic-search/src/AllFollowed.jsx

```diff
@@ -3,32 +3,32 @@ import styles from './TopicSearch.scss';
 import classNames from 'classnames';
 
 // transform like this => topic1, topic2 and topic3
-const arrayToSentence = matchingFollowedTopics => {
-	const topicsLength = matchingFollowedTopics.length;
+const arrayToSentence = followedSuggestions => {
+	const topicsLength = followedSuggestions.length;
 
 	if (topicsLength === 1) {
-		return <b>{ matchingFollowedTopics[0].name }</b>;
+		return <b>{ followedSuggestions[0].prefLabel }</b>;
 	} else {
-		return matchingFollowedTopics
+		return followedSuggestions
 		.map((topic, index) => {
 			if (index + 1 === topicsLength) {
 				// the last topic
-				return  <span key={ index }>and <b>{ topic.name }</b></span>
+				return  <span key={ topic.id }>and <b>{ topic.prefLabel }</b></span>
 			} else {
```

Comment:
Probably don't need the `else` block here as you're `returning` in the `if` statement above. Might make this block slightly cleaner?

Reply:
:open_mouth: No, don't need the `else` block! Thank you!